hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/ContextualRetriever.java

```diff
@@ -41,6 +51,41 @@ private ContextualRetriever() {
         throw new UnsupportedOperationException("Utility class only");
     }
 
+    /**
+     * Returns the account if it exists and is usable. A {@link HandleException} is thrown if the account is invalid. Note that this method should also work with account ID's that represent smart contracts
+     *
+     * @param accountId the ID of the account to get
+     * @param accountStore the {@link ReadableTokenStore} to use for account retrieval
+     * @param autoRenewConfig the {@link AutoRenewConfig} to use for determining account expiry validation
+     * @throws HandleException if any of the account conditions are not met
+     */
+    public static Account getIfUsable(
```

Comment:
We already have `getIfUsable`. We should probably rename as `getIfUsableAndValidExpiry` or something similar

Reply:
We do? I'm not seeing it anywhere on the current `develop`. There's already a `getIfUsable` for a token, but I don't see one for an account...?